core/git_mixins/branches.py

```diff
@@ -22,7 +22,9 @@ def get_branches(self):
         """
         Return a list of all local and remote branches.
         """
-        stdout = self.git("branch", "-a", "-vv", "--no-abbrev", "--no-color")
+        savvy_settings = sublime.load_settings("GitSavvy.sublime-settings")
+        sort_by_recent = savvy_settings.get("sort_by_recent_in_branch_dashboard")
+        stdout = self.git("branch", "-a", "-vv", "--no-abbrev", "--no-color", "--sort=-committerdate" if sort_by_recent else None)
         return (branch
```

Comment:
If the order of the branch list is only intended for the dashboard, the `sublime.load_settings` should not be called here. Instead, the argument of the call should be extended. And the setting call be fetch at [here](https://github.com/divmain/GitSavvy/blob/8cfeb94360d71e0b3f7b40ed9f130cfe10b23a90/core/interfaces/branch.py#L82-L82).

If your original idea is to sort the branch list in any conditions, I guess it will be a bit too ambitious.

```python
    def get_branches(self, sortby=None):
        """
        Return a list of all local and remote branches.
        """
        stdout = self.git(
            "branch", "-a", "-vv", "--no-abbrev", "--no-color",
            "--sort=-{}".formate(sortby) if sortby else None
            )
        return (branch
                for branch in (self._parse_branch_line(self, line) for line in stdout.split("\n"))
                if branch)
```